### DISCUSSION TITLE
After logout wiki.js is not redirecting to the login page.

### DISCUSSION THREAD

**Discussion by ganesh112700** [requarks/wiki#5826] [Help / Questions]
After logout wiki.js is not redirecting to the login page, it's redirecting to the home page, using nodejs also redirecting is not working.

**Note: Using page permission also can't able to restrict the page, user login time page permission is working, but after logout, all pages are showing.**

**Comment by ikun-404** [comment]
I have the same question. I don't know how to solve this problem. Can I hide the logout button?

**Comment by ikun-404** [comment]
The solution is to turn off the read permission of the guest's group
reference resources
https://github.com/requarks/wiki/discussions/4844
